sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -150,6 +342,36 @@ export class Sender {
     sendBatch(messages: SendableMessageInfo[]): Promise<void>;
     }
 
+// @public
+export class ServiceBusAtomManagementClient extends ServiceClient {
+    constructor(connectionString: string, options?: ServiceBusAtomManagementClientOptions);
+    createQueue(queueName: string, queueOptions?: QueueOptions): Promise<CreateQueueResponse>;
+    createRule(topicName: string, subscriptionName: string, ruleName: string, ruleOptions?: RuleOptions): Promise<CreateRuleResponse>;
+    createSubscription(topicName: string, subscriptionName: string, subscriptionOptions?: SubscriptionOptions): Promise<CreateSubscriptionResponse>;
+    createTopic(topicName: string, topicOptions?: TopicOptions): Promise<CreateTopicResponse>;
+    deleteQueue(queueName: string): Promise<DeleteQueueResponse>;
+    deleteRule(topicName: string, subscriptionName: string, ruleName: string): Promise<DeleteRuleResponse>;
+    deleteSubscription(topicName: string, subscriptionName: string): Promise<DeleteSubscriptionResponse>;
+    deleteTopic(topicName: string): Promise<DeleteTopicResponse>;
+    getQueueDetails(queueName: string): Promise<GetQueueResponse>;
+    getRule(topicName: string, subscriptioName: string, ruleName: string): Promise<GetRuleResponse>;
```

Comment:
Why no `Details` suffix to the `getRule` method name to be in line with the `getQueueDetails`, `getTopicDetails` and `getSubscriptionDetails`?

Reply:
This doesnt block this PR, and can be addressed after this PR is merged

Reply:
This doesnt block this PR, and can be addressed after this PR is merged